docs/release-notes/2.7.0.asciidoc

```diff
@@ -53,7 +57,6 @@
 * Update registry.access.redhat.com/ubi8/ubi-minimal Docker tag to v8.7-1085 {pull}6438[#6438]
 * Update module github.com/hashicorp/vault/api to v1.9.0 {pull}6403[#6403]
 * Update github.com/dgryski/go-lttb digest to f8fc36c {pull}6387[#6387]
-* Make TestRedClusterCanBeModifiedByDisablingPredicate more resilient with retries {pull}6370[#6370] (issue: {issue}6184[#6184])
```

Comment:
Added the [:ci](https://github.com/elastic/cloud-on-k8s/labels/%3Aci) label to ignore it.